editor/import/3d/scene_import_settings.cpp

```diff
@@ -1585,9 +1585,18 @@ void SceneImportSettingsDialog::_save_dir_confirm() {
 			continue; //ignore
 		}
 		String path = item->get_text(1);
+		String uid_path;
 		if (!path.is_resource_file()) {
 			continue;
 		}
+		if (path.begins_with("uid://")) {
+			uid_path = path;
+			path = ResourceUID::ensure_path(uid_path);
```

Comment:
```suggestion
			path = ResourceUID::uid_to_path(uid_path);
```
`ensure_path()` should be used when the original path can be either.